src/styles/catalog.scss

```diff
@@ -0,0 +1,25 @@
+.main {
+  margin: 0 auto;
```

Comment:
Don't set margins for BEM blocks, you can set additional class for this element for making it element of another BEM block